### DISCUSSION TITLE
Library management tools within the scope of the project

### DISCUSSION THREAD

**Discussion by Jdiesel87** [advplyr/audiobookshelf#372] [Ideas]
I didn't want to make a FR for this but rather start a discussion to gauge interest instead. The title isn't very clear but what I am getting at is what tools to manage the library from a user/admin would be useful?

Some thoughts:

- User reporting feature to tag a book as having an issue (missing/incorrect tracks, mislabeled book metadata, etc)
- Book request feature allowing users to request a book that isn't currently in the library
- User reviews/ratings to share feedback on a book with other users

What else would be useful? Does it make sense for it to be a core feature or better suited as a plugin or 3rd party app?

**Comment by Budlyte** [comment]
I like `User reporting feature to tag a book as having an issue (missing/incorrect tracks, mislabeled book metadata, etc)`, this is just helpful to everyone using a server.

`Book request feature allowing users to request a book that isn't currently in the library` - this isn't really in line with ABS. This is a library management & player tool, not a search and download manager. For this I suggest looking into Petio, Ombi, Overseer, etc and asking on their Discord channels that they support Readarr or LazyLibrarian.

`User reviews/ratings to share feedback on a book with other users` - I dunno. Maybe a link to the page on Goodreads, custom page on Discord, or something. There are spaces for this already though.
